Modules/_curses_panel.c

```diff
@@ -429,21 +429,30 @@ PyCursesPanel_New(_curses_panel_state *state, PANEL *pan,
     return (PyObject *)po;
 }
 
+static int
+PyCursesPanel_Clear(PyObject *op)
+{
+    PyCursesPanelObject *self = _PyCursesPanelObject_CAST(op);
+    PyObject *extra = (PyObject *)panel_userptr(self->pan);
+    if (extra != NULL) {
+        Py_CLEAR(extra);
```

Comment:
Py_DECREF is enough.

Reply:
I thought `tp_clear` could be called more than once. If this is the case, I would still need to really clear it right?